ViewGenerator/tools/webpack/Plugins/CommonConfiguration.ts

```diff
@@ -136,6 +136,7 @@ let getCommonConfiguration = (isProductionBuild: boolean, cacheName: string, lib
     
     if (isProductionBuild) {
         Configuration.optimization = {
+            flagIncludedChunks: false,
```

Comment:
why is this made false?

Reply:
From webpack documentation:

>Tells webpack to determine and flag chunks which are subsets of other chunks in a way that subsets don’t have to be loaded when the bigger chunk has been already loaded. By default optimization.flagIncludedChunks is enabled in production [mode](https://webpack.js.org/configuration/mode/) and disabled elsewise.

This needs to be false because each view is self-contained (shadow-dom) and we need to load the script for every view

Reply:
If this was set to true, there couldn't be reusal of code between views and we would get runtime errors